game-app/game-core/src/main/java/games/strategy/engine/framework/network/ui/SetMapClientAction.java

```diff
@@ -50,6 +46,7 @@ public void actionPerformed(final ActionEvent e) {
     if (name == null || name.length() <= 1) {
       return;
     }
-    serverStartupRemote.changeServerGameTo(name);
+    // don't block UI thread
```

Comment:
Rather than a comment - it might  be better documentation to have a 'precondition' check that the current thread is the EDT.

Reply:
That's a good idea